README.md

```diff
@@ -1,350 +1,61 @@
-    environment:
-      - DJANGO_SETTINGS_MODULE=hhs_oauth_server.settings.dev
-      - DJANGO_FHIR_CERTSTORE=/certstore
-      - DATABASES_CUSTOM=postgres://postgres:toor@bbdb:5432/bluebutton
-      - OAUTHLIB_INSECURE_TRANSPORT=true
-      - DJANGO_DEFAULT_SAMPLE_FHIR_ID="20140000008325"
-      - DJANGO_SECURE_SESSION=False
-      - DJANGO_MEDICARE_LOGIN_URI=http://127.0.0.1:8080?scope=openid%20profile&client_id=bluebutton
-      - DJANGO_SLS_USERINFO_ENDPOINT=http://msls:8080/userinfo
-      - DJANGO_SLS_TOKEN_ENDPOINT=http://msls:8080/token
-      - FHIR_URL=https://bfd.local:9954/v1/fhir/
-    ports:
-      - "8000:8000"
-    links:
-      - "bfd:bfd.local"
-    volumes:
-      - ${BB20_CONTEXT}:/code
-      - ${CERTSTORE}:/certstore
-    depends_on:
-      - bbdb
-```
-
-The BlueButton 2 system also requires an Identity Provider
-`docker-compose.msls.yml`
-```
-version: '3.3'
-
-services:
-
-  msls:
-    build:
-      context: ../../bb.dev/msls
-      dockerfile: Dockerfile
-    command: msls
-    ports:
-      - "8080:8080"
-```
-
-Bringing these systems up together:
-
-```
-docker-compose -f docker-compose.yml -f docker-compose.bb2.yml -f docker-compose.msls.yml up -d
-```
-
-```
-docker-compose -f docker-compose.bb2.yml exec bb2 ./docker-compose/migrate.sh
-```
-
-### Run Scripts
-
-The `apps/utils/scripts` directory contains bash scripts that simplify running the various components of BFD on a UNIX-like system.
-Refer to the README files in that directory for detailed information about the scripts.
-
-### Security
+## Security
 
 We work with sensitive information: do not put any PHI or PII in the public repo for BFD.
 
-If you believe you’ve found or been made aware of a security vulnerability, please refer to the CMS Vulnerability Disclosure Policy (here is a [link](https://www.cms.gov/Research-Statistics-Data-and-Systems/CMS-Information-Technology/CIO-Directives-and-Policies/Downloads/CMS-Vulnerability-Disclosure-Policy.pdf) to the most recent version as of the time of this commit.
-
-### Eclipse Configuration
-
-The following instructions are to be executed from within the Eclipse IDE application to ensure proper configuration.
-
-#### Eclipse JDK
-
-Verify Eclipse is using the correct Java 17 JDK.
-
-1. Open **Window > Preferences**.
-1. Select **Java > Installed JREs**.
-1. If your JDK does not appear in the **Installed JREs** table add it by clicking the **Add** button, select **Standard VM** and locate your installation using the **Directory...** button.
-1. Ensure your JDK is selected in the **Installed JREs** table by checking the checkbox next to the JDK you wish to use.
-
-#### Eclipse Preferences
+If you believe you’ve found or been made aware of a security vulnerability, please refer to 
+the CMS Vulnerability Disclosure Policy (here is a [link](https://www.cms.gov/Research-Statistics-Data-and-Systems/CMS-Information-Technology/CIO-Directives-and-Policies/Downloads/CMS-Vulnerability-Disclosure-Policy.pdf) 
```

Comment:
Noticed a spare '(' in the content here, and this is _merely_ a suggestion.
```suggestion
the [CMS Vulnerability Disclosure Policy](https://www.cms.gov/Research-Statistics-Data-and-Systems/CMS-Information-Technology/CIO-Directives-and-Policies/Downloads/CMS-Vulnerability-Disclosure-Policy.pdf)
```

Reply:
Makes sense, fits the style of the rest of the links. I had just copy/pasted that section without editing it.